ddtrace/internal/core/crashtracker_exe.py

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env python
```

Comment:
this will always get the right Python version?

e.g. a system that has `python`, `python3`, `python3.8`, etc... ?

Reply:
ah, that's another direction this could get into trouble. probably better to specify `python3` but this shebang line only works on Unix like systems and not on windows. 
Eventually we'd want to support crashtracker on windows too, then it looks more portable to install a command as I did before. What do you think?

Reply:
I still think if we can determine the Python executable currently running, we should just use it, seems like it would be the safest approach, no?

Reply:
Updated to `python3`.